tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.Iast.cs

```diff
@@ -24,6 +24,12 @@ internal class Iast
             /// Configuration key for controlling which weak cipher algorithms are reported.
             /// </summary>
             public const string WeakCipherAlgorithms = "DD_IAST_WEAK_CIPHER_ALGORITHMS";
+
+            /// <summary>
+            /// Configuration key for enabling or disabling the vulnerability duplication detection.
+            /// Default is value is true (enabled).
```

Comment:
```suggestion
            /// Configuration key for enabling or disabling the vulnerability duplication detection.
            /// When enabled, a vulnerability will only be reported once in the lifetime of an app,
            /// instead of on every usage. Default is value is true (enabled).
```